tests/components/ecovacs/conftest.py

```diff
@@ -139,7 +142,7 @@ async def init_integration(
     mock_mqtt_client: Mock,
     mock_device_execute: AsyncMock,
     platforms: Platform | list[Platform],
-) -> MockConfigEntry:
+) -> AsyncGenerator[MockConfigEntry, None]:
```

Comment:
Why is this asyncGenerator instead of Generator?

Reply:
Async function